BrainCloudClient/Assets/BrainCloud/Client/BrainCloud/BrainCloudClient.cs

```diff
@@ -174,9 +174,9 @@ public class BrainCloudClient
         private BrainCloudChat _chatService;
         private BrainCloudRTT _rttService;
         private BrainCloudRelay _rsService;
-
+        
```

Comment:
🚲 
This file was not touched. Those are pointless changes that will add your name to the git-blame. Can make researching for issues in the future more difficult "oh, franco touched that file around that time! Oh no, he did nothing"
:)